packages/v4/src/content/design-guidelines/components/modal/modal.md

```diff
@@ -7,111 +7,131 @@ related: ['Button']
+<img src="./img/multi-step-destructive-confirmation-dialog.png" alt="A multi-step destructive confirmation dialog's button activates only after a user types DELETE into the input field." />
 
-Depending on your design template, follow this format for confirmation dialogs:
-* **Headline:** A headline is usually phrased as a question. Include important keywords (like “permanent”) in the headline.
-* **Body text:** Body text gives information about the action’s consequence.
-* **Buttons:** Buttons allow a user to answer the headline question. These can simply be “yes” or “no,” but adding some context increases clarity—especially when skimming.
 
-**Note:** Since confirmation dialogs disrupt user task flow, do not provide confirmation dialogs for actions that are easily reversed or insignificant.
+### Error dialogs
 
-##### Confirm a non-destructive action
-<img src="./img/confirm-non-destructive-action.png" alt="Confirm a non-destructive action" />
+<img src="./img/error-dialog.png" alt="An example of a successful error dialog, with and without an error icon" />
 
-Provide buttons on confirmation dialogs that make it simple for users to understand and confirm their choices. When choosing button labels, use descriptive verbs that clearly indicate the consequence of a selection, like Delete, Save, or Create. Avoid vague labels like OK or Done.
+Use an error dialog to inform users of problems that interrupt normal or expected behavior. Briefly contextualize the problem and why it happened, then provide actionable steps toward a solution.
 
-Ambiguous title and button labels make it more difficult for the user to understand their options and make a decision. Descriptive title and button labels make it easy for the user to scan the content and make a decision quickly.
+Learn more about writing error messages in our [error message guidelines](/ux-writing/error-messages).
 
-##### Confirm a destructive action
-<img src="./img/confirm-destructive-action.png" alt="Confirm a destructive action" />
+Error dialogs may use an error icon for visual emphasis.
 
-If an action is difficult or impossible to undo, a warning icon can be incorporated into the modal to add emphasis. If an action will seriously impact the system and cannot be undone, use a destructive style button. Otherwise, use a primary button.
+### Passive dialogs
 
-Sometimes it is necessary to require an additional confirmation with a destructive action. An example using a form field can be seen below. In order for the destructive action to become actionable, the user needs to type in a word or phrase to make the button active.
-<img src="./img/2-step-confirm-destructive-action.png" alt="2 step confirm a destructive action" />
+<img src="./img/passive-dialog.png" alt="An example of a successful passive dialog, with and without a warning icon" />
```

Comment:
Spacing between body and buttons seems bigger than necessary! Would double check that, as well as the button alignment with the text